packages/@aws-cdk/cloud-assembly-schema/lib/schema.ts

```diff
@@ -1,7 +1,25 @@
 import { ArtifactProperties } from './artifact-schema';
+import { ManifestFile } from './assets';
 import { ContextProvider, ContextQueryProperties } from './context-queries';
 import { MetadataEntry } from './metadata-schema';
 
+/**
+ * Schemas for various manifests within the cloud assembly.
+ */
+export interface Assembly {
```

Comment:
Can we move the `version` property to this level?

Reply:
`Assembly` is a fictional structure. It does not have a physical representation. I don't see value in moving version here before it's never loaded from disk.

Reply:
I mistakenly thought we switched to having 1 file that embeds both schemas. But we actually have two files, `manifest.json` and `assets.json` - right?

Reply:
Something still feels off. We now have a schema definition file that doesn't actually match any of our files. This means that validating `manifest.json` against the new schema is not going to work, which seems inherently wrong. Users would have to do this `{ manifest: loadManifest() }` hack as well. 

Also, having them both optional would basically mean that any file will be validated successfully, no? 

Honestly it feels we should either consolidate the two files into an `assembly.json` file, or have two separate schemas that are simply versioned together.

Reply:
Okay, decoupled the two schemas.